interfaces/cython/cantera/examples/thermo/isentropic.py

```diff
@@ -69,5 +69,7 @@ def isentropic(gas=None):
         plt.show()
 
     except ImportError:
+        print('Matplotlib not found. Unable to plot results.')
+    except:  # noqa
```

Comment:
The current modification would result in data no longer being displayed, i.e. it breaks current behavior.

Reply:
It only falls back to that behavior if an error outside of ImportError is encountered. The current behavior needs to be more specific if Matplotlib isnt installed.

Reply:
I believe it was the intent of the original implementation to display results if matplotlib is not installed. The way you changed this, this no longer gets displayed. I.e. things get displayed if there is another error. Before your change, other errors are raised (which is likely by design).

Reply:
I agree with @ischoegl's interpretation - there's no need to add the extra bare `except` statement here. Adding the `print()` call is fine though.

Reply:
Ah I see. Okay well I can close this PR then.